code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1693,6 +1693,17 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	glass_name = "Mojito"
 	glass_desc = "A drink that looks as refreshing as it tastes."
 
+/datum/reagent/consumable/ethanol/moscow_mule
+	name = "Moscow Mule"
+	description = "A chilly drink that reminds you of the Derelict."
```

Comment:
![image](https://user-images.githubusercontent.com/33292112/71639854-3d5a0580-2c7f-11ea-8347-d7310cde70c8.png)
Not sure I have ever seen the derelict referred to as such in game, its either KS13 or KC13.

Reply:
> ![image](https://user-images.githubusercontent.com/33292112/71639854-3d5a0580-2c7f-11ea-8347-d7310cde70c8.png)
> Not sure I have ever seen the derelict referred to as such in game, its either KS13 or KC13.

C is just russian s

Reply:
I’m interested to hear what others think, but it was my thought that saying “the Derelict” alludes more to myth than reality as opposed to saying KC13 (which as we know, never existed and certainly was never associated with NT). Anyways, I’m up for changing this flavor text, I just thought it would be cute.

Reply:
was this okayed? If you didn't change anything don't resolve.

Reply:
KC 13 definitely exists?

Reply:
the sign is a joke, potentially a "there is no cow level" reference.

Reply:
> was this okayed? If you didn't change anything don't resolve please.

Whoops, did not mean to resolve.

Specifically, whose okay should I be getting on this?

Would it be better if it said “the myth of the Derelict” or even “a rumor you heard about a derelict station in space”? I specifically wanted to allude to the myth or a Russian station without calling it by name - because the lore is that KC13 was a secret and NT denies any knowledge of it? So I don’t see the average Joe knowing its name.

Of course, the reference could be removed entirely, it just seemed apt to reference a Russian station with the Moscow Mule.

Reply:
It looks like it is referred to as such in the whiteship navigation console.